controllers/cache/utils.go

```diff
@@ -36,6 +36,7 @@ func generateName(typeName, metaName string) string {
 	return fmt.Sprintf("%s-%s", typeName, metaName)
 }
 
+// Deprecated
```

Comment:
So @Ghostbaby should replace this function with the new common util func?

Reply:
yes, it moved to common util.

Reply:
ok, I will remove this function.